benchmark_utils/metrics.py

```diff
@@ -2,6 +2,7 @@
 
 with safe_import_context() as import_ctx:
     import numpy as np
+    from sklearn.metrics import f1_score
```

Comment:
But if we depend on `scikit-learn`, we should update the `objective.py` file to add `requirements = ['scikit-learn']` to make the benchmark installable.